torch/linalg/__init__.py

```diff
@@ -82,6 +82,64 @@
     True
 """)
 
+inv = _add_docstr(_linalg.linalg_inv, r"""
+linalg.inv(input, *, out=None) -> Tensor
+
+This function computes the "multiplicative inverse" matrix of a square matrix, or batch of such matrices, :attr:`input`.
+The result satisfies the relation
```

Comment:
Looking at the rendered doc (https://9738771-65600975-gh.circle-artifacts.com/0/docs/linalg.html) I think we should separate this expression as a mathematical expression on its own line, and just write "float" and "cfloat" and etc. normally to prevent this brief grey bits cluttering the documentation.

We should probably do the same with "None" below, too.

Reply:
I updated the documentation.
Circle ci job for docs doesn't have the saved artifacts though https://app.circleci.com/pipelines/github/pytorch/pytorch/254406/workflows/71732683-eecf-45d9-87e3-cf08161c089d/jobs/9812964/artifacts

Reply:
You have to look in the python_doc_build: https://9822947-65600975-gh.circle-artifacts.com/0/docs/linalg.html.

<img width="841" alt="Screen Shot 2020-12-22 at 9 41 33 PM" src="https://user-images.githubusercontent.com/38511765/102963645-7bf35c00-449e-11eb-82fb-c7452f57af75.png">

We want to avoid the grey broken up by white effect here.

Reply:
Probably this could be fixed by tweaking sphinx configuration for whitespace significance in inline code blocks.

Reply:
Interesting, I've never seen that. Maybe this effect kicks in when some expression is above a certain length. Anyway I'd write this as:
```
The result satisfies the relation::

    matmul(inv(input), input) = matmul(input, inv(input)) = eye(input.shape[0]).expand_as(input)
```
because it's not meant as display rather than inline (you don't want a line break in the expression).

Reply:
I think the "math" header will render this correctly, like here:

https://github.com/pytorch/pytorch/blob/5171bd94d7b39ad683e1b6b4f883e85a919175a7/torch/_torch_docs.py#L131

In the doc for abs:

https://pytorch.org/docs/master/generated/torch.abs.html?highlight=abs#torch.abs